src/scenes/ProgressReports/EditForm/Steps/Media/ImageField.tsx

```diff
@@ -88,15 +91,31 @@ export const ImageField = ({
         ...extendSx(props.sx),
       ]}
     >
-      <Box
-        component="img"
-        src={current.url}
-        crossOrigin="use-credentials"
-        sx={{
-          maxWidth: 1,
-          borderRadius: 1,
-        }}
-      />
+      {current.__typename === 'Image' ? (
+        <Box
+          component="img"
+          src={current.url}
+          crossOrigin="use-credentials"
+          sx={{
+            maxWidth: 1,
+            borderRadius: 1,
+          }}
+        />
+      ) : (
+        <Card
+          elevation={5}
```

Comment:
The Card within the accordion "card" looks weird. Maybe this can just be an outlined variant?

Reply:
Yup. I was going back and forth with this one. I liked the outline varian, but the border color didn't match the fields borders color.